src/main/java/TaskList.java

```diff
@@ -0,0 +1,167 @@
+public class TaskList {
+    private Task[] taskArray = new Task[100];
+    private int totalTaskNum = 0;
+
+    public boolean addTask(String userInput){
```

Comment:
Might be good to  document the functions using params learned in cs1010, would improve readability